classes/submissionFile/Repository.inc.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @file classes/submissionFile/Repository.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class submissionFile
+ *
+ * @brief A repository to find and manage submission files.
+ */
+
+namespace APP\submissionFile;
+
+use APP\core\Application;
+use Exception;
+use PKP\db\DAORegistry;
+use PKP\search\SubmissionSearch;
+use PKP\submissionFile\Repository as BaseRepository;
+use PKP\submissionFile\SubmissionFile;
+
+class Repository extends BaseRepository
+{
+    public function add(SubmissionFile $submissionFile): int
+    {
+        $submissionId = parent::add($submissionFile);
```

Comment:
This should be `$submissionFileId`, not `$submissionId` (here and below), right?